src/test/java/org/folio/rest/api/PatronNoticePoliciesApiTest.java

```diff
@@ -296,6 +296,144 @@ public void cannotDeleteInUsePatronNoticePolicy() throws InterruptedException, M
     assertThat(message, is("Cannot delete in use notice policy"));
   }
 
+  @Test
+  public void cannotUpdatePatronNoticePolicyWithNotValidSendHow() throws InterruptedException,
+    MalformedURLException, TimeoutException, ExecutionException {
+
+    JsonObject sendOptions = new JsonObject()
+      .put("sendWhen", "Renewed")
+      .put("sendHow", "Before");
+
+    JsonObject loanNotice = new JsonObject()
+      .put("templateId", UUID.randomUUID().toString())
+      .put("format", "Email")
+      .put("realTime", "true")
+      .put("sendOptions", sendOptions);
+
+    JsonObject noticePolicy = createNoticePolicyWithLoanNotice(loanNotice);
+    JsonResponse response = putPatronNoticePolicy(noticePolicy);
+    assertThat(response.getStatusCode(), is(422));
+    assertThat(response.getJson().getJsonArray("errors").getJsonObject(0).getString("message"),
+      is("sendHow should not be present"));
```

Comment:
Please change error messages to be more informative

Reply:
Thank you for review, updated